doc/ChangeLog

```diff
@@ -1,4 +1,71 @@
 ===============================================================
+Tag name: ctsm5.3.073
+Originator(s): slevis (Samuel Levis,UCAR/TSS,303-665-1310)
+Date: Tue 26 Aug 2025 03:20:16 PM MDT
+One-line Summary: Update .gitmodules to cesm3_0_alpha07c
+
+Purpose and description of changes
+----------------------------------
+ Update ccs_config_cesm1.0.48 to ccs_config_cesm1.0.56.
+ Update cime6.1.112 to cime6.1.113.
+ Changes answers in gnu and nvhpc tests on derecho.
+
```

Comment:
Suggest adding something like the following:

```suggestion
ccs_config1.0.49 -- Moves the NVHPC compiler version back a bit on Derecho which causes changes in answers
ccs_config1.0.52 -- Update the GNU compiler version on Derecho, which causes a change in answers
```

Reply:
Done: Thank you for this suggestion @ekluzek